lib/g10_certification_test_kit/bulk_data_authorization.rb

```diff
@@ -0,0 +1,229 @@
+require_relative 'authorization_request_builder'
+
+module G10CertificationTestKit
+  class BulkDataAuthorization < Inferno::TestGroup
+    title 'Bulk Data Authorization'
+    description <<~DESCRIPTION
+      Bulk Data servers are required to authorize clients using the
+      [Backend Service Authorization](http://hl7.org/fhir/uv/bulkdata/STU1/authorization/)
+      specification as defined in the [FHIR Bulk Data Access IG v1.0.0](http://hl7.org/fhir/uv/bulkdata/STU1/).
+
+      In this set of tests, Inferno serves as a Bulk Data client that requests authorization
+      from the Bulk Data authorization server.  It also performs a number of negative tests
+      to validate that the authorization service does not improperly authorize invalid
+      requests.
+
+      This test returns an access token.
+    DESCRIPTION
+
+    id :bulk_data_authorization
+
+    input :bulk_token_endpoint,
+          title: 'Backend Services Token Endpoint',
+          description: 'The OAuth 2.0 Token Endpoint used by the Backend Services specification
+                        to provide bearer tokens.',
+          default: 'https://inferno.healthit.gov/bulk-data-server/auth/token'
+    input :bulk_client_id,
+          title: 'Bulk Data Client ID',
+          description: 'Client ID provided at registration to the Inferno application.'
```

Comment:
```suggestion
          description: 'Client ID provided at registration to the Inferno application.'
          default: 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6InJlZ2lzdHJhdGlvbi10b2tlbiJ9.eyJqd2tzX3VybCI6Imh0dHBzOi8vaW5mZXJuby5oZWFsdGhpdC5nb3YvaW5mZXJuby8ud2VsbC1rbm93bi9qd2tzLmpzb24iLCJhY2Nlc3NUb2tlbnNFeHBpcmVJbiI6MTUsImlhdCI6MTU5OTE1NzgyMX0.-wulnE05BlY_Zcm5iP77Meqxr6iNiYxBsOADB5CGE8I'
```
Can we bring over the default from Program